xbmc/music/MusicDatabase.cpp

```diff
@@ -753,7 +753,7 @@ bool CMusicDatabase::GetSong(int idSong, CSong& song)
     {
       const dbiplus::sql_record* const record = m_pDS.get()->get_sql_record();
 
-      int idSongArtist = record->at(songArtistOffset + artistCredit_idArtist).get_asInt();
+      record->at(songArtistOffset + artistCredit_idArtist).get_asInt();
```

Comment:
If the value is really not used then there's no need for the whole call as it's only purpose is to return a value for usage :) @DaveTBlake for checking why the value is not used.